enterprise/server/cmd/ci_runner/main.go

```diff
@@ -2248,13 +2314,18 @@ func runBashCommand(ctx context.Context, cmd string, env map[string]string, dir
 		return "", &commandError{err, ""}
 	}
 
+	return runCommandWithOutput(ctx, "bash", []string{"-eo", "pipefail", "-c", cmd}, env, dir, outputSink)
+}
+
+func runCommandWithOutput(ctx context.Context, executable string, args []string, env map[string]string, dir string, outputSink io.Writer) (string, *commandError) {
 	var buf bytes.Buffer
 	w := io.MultiWriter(outputSink, &buf)
 
-	if err := runCommand(ctx, "bash", []string{"-eo", "pipefail", "-c", cmd}, env, dir, w); err != nil {
+	if err := runCommand(ctx, executable, args, env, dir, w); err != nil {
 		return "", &commandError{err, buf.String()}
 	}
 	output := buf.String()
+	output = strings.ReplaceAll(output, "\x1b[0m", "")
```

Comment:
this seems like a bit of a smell, can this be removed?

Reply:
This is the ANSI reset char. We needed this when parsing out the workspace path from command output. Now that I refactored that, we can stop doing this. It doesn't matter when we're just logging the command output to the screen